src/Hashids.php

```diff
@@ -79,7 +100,10 @@ class Hashids implements HashidsInterface
      *
      * @return void
      */
-    public function __construct($salt = '', $minHashLength = 0, $alphabet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890')
+    public function __construct(
+        $salt = Hashids::DEFAULT_SALT,
```

Comment:
@roukmoute I think we cannot use `static` in the default parameters due to language limitations. It should be either `self` or `Hashids`. Alternatively we can have all parameters as `null` by default, and set them as `static` in the constructor body.